src/test/java/com/epam/tc/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,205 @@
+package com.epam.tc.hw2.ex1;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class HomePageTest {
+
+    private WebDriverWait wait;
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void prepareDriver() {
+        driver = Utils.getChromeDriver();
+        driver.manage().timeouts().pageLoadTimeout(30, TimeUnit.SECONDS);
+        wait = new WebDriverWait(driver, 10);
+    }
+
+    // 12. Close Browser
+    @AfterMethod
+    private void close() {
+        driver.quit();
+    }
+
+    @Test
+    public void testHomePage() {
+        // 1. Open test site by URL
+        openSite();
+
```

Comment:
SoftAssert is created in each of the methods, but I think would be better to create it once in the `@Test` method and pass it  to subsequent calls